modules/artifact/src/main/java/org/mule/runtime/module/artifact/classloader/ArtifactClassLoaderFilter.java

```diff
@@ -70,9 +87,9 @@ public boolean exportsClass(String className)
     public boolean exportsResource(String name)
     {
         checkArgument(name != null, "Resource name cannot be null");
-        final String resourcePackage = getResourceFolder(name);
+        final String sanitizeResourceName = sanitizeResourceName(name);
 
-        return exportedResourcePackages.contains(resourcePackage);
+        return exportedResources.contains(sanitizeResourceName);
```

Comment:
Wouldn't you need to change all the mule-module.properties to start exporting files instead of folders? For instance, I'm looking at core and it is exporting:
```
artifact.export.resourcePackages=/,\
                                /META-INF,\
                                /META-INF/services,\
                                /META-INF/services/org/mule/runtime/core/config,\
                                /META-INF/services/org/mule/runtime/core/i18n,\
                                /META-INF/services/org/mule/runtime/core/models,\
                                /META-INF/services/org/mule/runtime/transport
```
Perhaps I' m missing something here but it should be explicitly exported the file if not it won't return true here. Right?